README.md

```diff
@@ -39,7 +39,10 @@
 - Vscode
 
```

Comment:
- [ ] In your README documentation, it may be of great help to users if you add commands on how to install packages and run the app ie. `npm install` and `npm start`. Kindly add these commands.